pkg/adapter/adapter.go

```diff
@@ -302,56 +305,127 @@ func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 	}
 }
 
-func (a *Adapter) postMessage(msg wabbit.Delivery) error {
-	a.logger.Info("url ->" + a.httpMessageSender.Target)
-	req, err := a.httpMessageSender.NewCloudEventRequest(a.context)
-	if err != nil {
-		return err
+func msgContentType(msg wabbit.Delivery) (string, error) {
+	var contentType string
+
+	for key, val := range msg.Headers() {
+		if strings.ToLower(key) == "content-type" {
+			contentType = strings.ToLower(fmt.Sprint(val))
+			break
+		}
+	}
+
+	if contentType == "" {
+		return "", fmt.Errorf("no content type present on Rabbitmq msg")
```

Comment:
I think instead of fmt.Errorf you might prefer to just do errors.New since there's no formatting done here.